docs/pipe/installation.rst

```diff
@@ -7,28 +7,51 @@ To install PIPE, run::
     cd PIPE
     python setup.py install
 
-PIPE uses python3 and requires the following modules: numpy, scipy, astropy
+PIPE uses python3 and requires the following packages: numpy, scipy, astropy
 
 Required file structure
 +++++++++++++++++++++++
 
-PIPE assumes the following directory structure:
+PIPE uses astropy's configuration machinery to manage for your default data and
+reference library locations. By default, you will find an empty directory in
 
-Data from DACE is to be put in ``data_root/name/visit/``.
-Output will then be put into ``data_root/name/visit/Outdata/version/``
-where "version" is a number. Multiple versions can be generated in the Outdata
-directory, e.g. from multiple runs with different input parameters.
+.. code-block:: python
 
-Reference files are put into ``ref_lib_path/``
-E.g., Gain correction file: ``CH_TU2020-02-18T06-15-13_REF_APP_GainCorrection_V0109.fits``
-Flat field library: ``flats.fits``
-Non-linearity correction: ``nonlin.txt``
+    >>> from pipe.config import cache_dir
+    >>> print(cache_dir)  # doctest: +SKIP
+    '~/.astropy/cache/.pipe-cheops'
 
-Generated PSF-files are put into a subdirectory: ``ref_lib_path/psf_lib/``
+Data from DACE should be placed in ``cache_dir``, in a subdirectory called:
+``data_root/name/visit/``, where ``name`` is the name of your target, and ``visit``
+is an index for the visit number. Output will then be saved to
+``data_root/name/visit/Outdata/version/`` where "version" is a number. Multiple
+versions can be generated in the ``Outdata`` directory, e.g. from multiple runs
+with different input parameters.
 
-as are ``eigenlib``, libraries of principal PSF components, also generated.
+Reference files are put into a subdirectory of ``cache_dir`` called
+``ref_lib_data/``. These files might include: the gain correction file:
+``CH_TU2020-02-18T06-15-13_REF_APP_GainCorrection_V0109.fits``, the flat field
+library: ``flats.fits``, and the non-linearity correction: ``nonlin.txt``.
 
-``data_root`` and ``ref_lib_path`` are defined in pipe_config.py.
+PSF-files that are constructed by PIPE will be stored within ``cache_dir`` in
+``ref_lib_data/psf_lib/``, as are ``eigenlib``, libraries of principal PSF
+components, also generated.
+
+If you would like to direct PIPE towards different locations for the
+``data_root`` and ``ref_lib_data`` directories temporarily (within the specific
+Python instance you're currently running), you can set them by setting
+the following variables:
+
+.. code-block:: python
+
+    from pipe import conf
+
+    conf.data_root = 'path/to/data_root'
+    conf.ref_lib_data = 'path/to/ref_lib_data
+
+If you'd like to set the path to the ``data_root`` and ``ref_lib_data``
+directories more permanently, for each and every time PIPE accesses them, you
+can edit the paths in ``pipe/config/conf.json`` and run ``python setup.py install``.
```

Comment:
This (may) be closer to the change you're looking for. It does require that you run `python setup.py install` to get changes to `pipe/config/conf.json`.